src/main/scala/DemoFortis.scala

```diff
@@ -278,6 +314,19 @@ object DemoFortis {
           )
         )
       ),
+      "radio" -> List(
+        ConnectorConfig(
+          "Radio",
```

Comment:
Can you provide an example with streaming from more than one source, just to prove out that we can support a list of audio sources.